app/code/Magento/LayeredNavigation/Test/Unit/Block/NavigationTest.php

```diff
@@ -5,7 +5,7 @@
  */
 
 namespace Magento\LayeredNavigation\Test\Unit\Block;
-
+use Magento\Catalog\Model\Category;
```

Comment:
There MUST be one blank line after the namespace declaration, and there MUST be one blank line after the block of use declarations. https://www.php-fig.org/psr/psr-2/